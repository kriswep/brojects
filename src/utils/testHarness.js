export default window.matchMedia = window.matchMedia || function() {
    return {
        matches : false,
        addListener : () => true,
        removeListener: () => true,
    };
};