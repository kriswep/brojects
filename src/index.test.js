import './utils/testHarness';
import React from 'react';
// import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Index, { IndexComponent } from './index.js';
// import App from './containers/App';

describe('index', () => {
  it('renders without crashing', () => {
    expect(JSON.stringify(
      Object.assign({}, Index, {_reactInternalInstance: 'censored'})
      )).toMatchSnapshot();
  });
  it('shallow renders without crashing', () => {
    // expect(JSON.stringify(
    //   Object.assign({}, Index, {_reactInternalInstance: 'censored'})
    //   )).toMatchSnapshot();
    const wrapper = shallow(<IndexComponent />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
