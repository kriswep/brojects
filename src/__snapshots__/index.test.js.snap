exports[`index renders without crashing 1`] = `"{\"props\":{\"store\":{},\"children\":{\"key\":null,\"ref\":null,\"props\":{},\"_owner\":null,\"_store\":{}}},\"context\":{},\"refs\":{},\"updater\":{},\"store\":{},\"_reactInternalInstance\":\"censored\",\"state\":null}"`;

exports[`index shallow renders without crashing 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }>
  <Connect(App) />
</Provider>
`;
